indico/modules/events/editing/client/js/management/editable_type/TeamManagerModal.jsx

```diff
@@ -0,0 +1,93 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+import principalsURL from 'indico-url:event_editing.api_editable_type_principals';
+
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Form as FinalForm} from 'react-final-form';
+import {Button, Form, Loader, Message, Modal} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+import {FinalPrincipalList} from 'indico/react/components';
+import {getChangedValues, FinalSubmitButton, FinalUnloadPrompt} from 'indico/react/forms';
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+import {useFavoriteUsers, useIndicoAxios} from 'indico/react/hooks';
+import {useNumericParam} from 'indico/react/util/routing';
+
+export default function TeamManager({editableType, onClose}) {
+  const eventId = useNumericParam('confId');
+
+  const {data: principals, loading: isLoadingPrincipals} = useIndicoAxios({
+    url: principalsURL({confId: eventId, type: editableType}),
+    trigger: eventId,
```

Comment:
`trigger: [eventId, editableType]`

While we don't ever expect this to happen, the editable type changing is something that would require a re-fetch, so like in the dependency list of other hooks it needs to be included here.